app/Policies/ContactPolicy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Domain\Auth\Models\UserModel;
+use App\Domain\Contacts\Models\ContactModel;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+final class ContactPolicy
+{
+    use HandlesAuthorization;
+
+    public function viewAny(UserModel $userModel): bool
+    {
+        return true;
+    }
+
+    public function view(UserModel $userModel, ContactModel $contactModel): bool
+    {
+        return true;
+    }
+
+    public function create(UserModel $userModel): bool
+    {
+        return true;
+    }
+
+    public function update(UserModel $userModel, ContactModel $contactModel): bool
+    {
+        return true;
+    }
+
+    public function delete(UserModel $userModel, ContactModel $contactModel): bool
+    {
+        return true;
+    }
+
+    public function restore(UserModel $userModel, ContactModel $contactModel): bool
```

Comment:
You can still `replicate()` a deleted contact. This will give a unique contraint eror when trying to save it, so it isn't a major issues.
But to me it looks more clean to also block replication, if you block restoring.

Reply:
I was wondering, how do you know exactly which methods you can implement on the policy? I noticed that these policies do not use interfaces. Is there a list of possible methods somewhere? I checked the docs but it seems there is no complete list documented there (there is no mention of `replicate` for example) https://laravel.com/docs/9.x/authorization#creating-policies

Reply:
For a Nova specific list: https://nova.laravel.com/docs/4.0/resources/authorization.html#undefined-policy-methods